src/Microsoft.AspNet.Mvc.TagHelpers/ScriptTagHelper.cs

```diff
@@ -243,36 +262,43 @@ private void BuildFallbackBlock(IDictionary<string, string> attributes, DefaultT
                        .Append(FallbackTestExpression)
                        .Append("||document.write(\"");
 
+                // May have no "src" attribute in the dictionary e.g. if Src and SrcInclude were not bound.
+                if (!attributes.ContainsKey(SrcAttributeName))
+                {
+                    // Need this entry to place each fallback source.
+                    attributes.Add(SrcAttributeName, null);
+                }
+
                 foreach (var src in fallbackSrcs)
                 {
-                    builder.Append("<script");
+                    // "src" values come from bound attributes and globbing. Must always be non-null.
```

Comment:
right. that takes its inputs from bound attributes and performs globbing on them.  further `FallbackSrc` isn't globbed at all.  the comment is correct.